crates/kornia-image/Cargo.toml

```diff
@@ -15,3 +15,11 @@ version.workspace = true
 kornia-tensor = { workspace = true }
 num-traits = { workspace = true }
 thiserror = { workspace = true }
+
+# Optional dependencies for arrow integration
+# NOTE: now we pin the arrow version to 54.2.1 because the latest version is not compatible with the dora-node-api crate
+arrow = { version = "54.2.1", optional = true }
```

Comment:
@haixuanTao are you guys planning to upgrade the min arrow version in dora at some point ?